src/claimtrie.cpp

```diff
@@ -1357,7 +1365,8 @@ bool CClaimTrieCache::insertClaimIntoTrie(const std::string& name, CClaimValue c
     {
         fChanged = true;
         currentNode->insertClaim(claim);
-        base->addToClaimIndex(name, claim);
+        CClaimIndexElement element = { name, claim };
```

Comment:
Is this working around a problem in the base class? Why not fix it there?

Reply:
No.